tools/dependencies-report/src/main/resources/licenseMapping.csv

```diff
@@ -33,6 +33,7 @@ dependency,dependencyUrl,licenseOverride
 "com.google.googlejavaformat:google-java-format:1.1",https://github.com/google/google-java-format,Apache-2.0
 "com.google.guava:guava:22.0",https://github.com/google/guava,Apache-2.0
 "com.google.j2objc:j2objc-annotations:1.1",https://github.com/google/j2objc/,Apache-2.0
+"commons-codec:commons-codec:1.10.0",http://commons.apache.org/proper/commons-codec/,Apache-2.0
```

Comment:
do we really have both here?

Reply:
I think I got confused by looking at the output of an older build.